src/core/util/threadpool.cpp

```diff
@@ -95,9 +100,22 @@ ThreadPool::Worker::Worker(ThreadPool& pool)
                 pool.tasks.pop();
             }
             state = State::Working;
-            task();
+            try {
+                task();
```

Comment:
Will this not silently ignore any exception thrown? Shouldn't we at least log an error?

Reply:
In principle that should never happen... it's mostly to be a bit defensive, if one missuses enqueueRaw...

Reply:
Probably should add a comment about that